packages/server/lib/util/print-run.ts

```diff
@@ -515,7 +515,7 @@ export function displayVideoProcessingProgress (opts: { videoName: string, video
         const dur = `${humanTime.long(finished)}`
 
         const table = terminal.table({
-          colWidths: [3, 21, 61, 15],
+          colWidths: [3, 22, 60, 15],
```

Comment:
had to offset the 2nd column by one in order to not get `finished compressin...` and not get the `g` truncated. I took a character from the 3rd column as we have more than ample room to store the results of the compression, but means the snapshots will be missing a space in that column (hence the regen in the system tests)